core/src/tile/tile.cpp

```diff
@@ -135,7 +135,7 @@ void Tile::draw(const Style& _style, const View& _view) {
         float zoomAndProxy = m_proxyCounter > 0 ? -m_id.z : m_id.z;
 
         shader->setUniformMatrix4f("u_model", m_modelMatrix);
-        shader->setUniformf("u_tile_origin", m_tileOrigin.x, m_tileOrigin.y, zoomAndProxy);
+        shader->setUniformf("u_tile_origin", m_tileOrigin.x, m_tileOrigin.y, m_id.z);
```

Comment:
I did some git history cleaning yesterday. Will test it again today and
confirm. Thanks
On Jan 12, 2016 6:00 AM, "Karim Naaji" <notifications@github.com> wrote:

> In core/src/tile/tile.cpp
> <https://github.com/tangrams/tangram-es/pull/458#discussion_r49439912>:
>
> > @@ -135,7 +135,7 @@ void Tile::draw(const Style& _style, const View& _view) {
> >          float zoomAndProxy = m_proxyCounter > 0 ? -m_id.z : m_id.z;
> >
> >          shader->setUniformMatrix4f("u_model", m_modelMatrix);
> > -        shader->setUniformf("u_tile_origin", m_tileOrigin.x, m_tileOrigin.y, zoomAndProxy);
> > +        shader->setUniformf("u_tile_origin", m_tileOrigin.x, m_tileOrigin.y, m_id.z);
>
> Can we remove this last param usage from the shader for no confusion with
> previous technique.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/458/files#r49439912>.
>